src/resample/tests/test_bootstrap.py

```diff
@@ -1,16 +1,7 @@
 import numpy as np
-from numpy.testing import assert_almost_equal, assert_equal
 import pytest
 
-from resample.bootstrap import (
-    bootstrap,
-    bootstrap_ci,
-    empirical_influence,
-    jackknife,
-    jackknife_bias,
-    jackknife_bias_corrected,
-    jackknife_variance,
-)
+from resample.bootstrap import bootstrap, bootstrap_ci, empirical_influence
```

Comment:
Question of whether empirical_influence belongs in jackknife or bootstrap (it's using the jackknife currently)

Reply:
I don't know much about empirical influence, looking at https://en.wikipedia.org/wiki/Robust_statistics#Empirical_influence_function, it seems that this is specific to the Jackknife, meaning that one cannot compute it with the general bootstrap, right? I think it should live in the jackknife module.

Reply:
It is ok if the bootstrap or the jackknife module have additional functions that are specific to them. Only the functions that are common between the two approaches should be named the same and have compatible calling signatures (as much as possible, bootstrap methods accept rng in addition and other parameters).